extra-multimedia/libbluray/autobuild/defines

```diff
@@ -8,5 +8,5 @@ BUILDDEP__LOONGSON3=""
 PKGBREAK="ffmpeg<=3.4.2 gvfs<=1.36.0 kodi<=1:17.4 \
           mplayer<=1:1.3.0-8 vlc<=3.0.1 xine-lib<=1.2.8-1"
 
-AUTOTOOLS_AFTER="--disable-examples --with-java9"
-AUTOTOOLS_AFTER__LOONGSON3="--disable-examples --without-java9 --disable-bdjava-jar"
+AUTOTOOLS_AFTER="--with-java9"
+AUTOTOOLS_AFTER__LOONGSON3="--without-java9 --disable-bdjava-jar"
```

Comment:
@liushuyu Could this possibly be enabled for Loongson 3, now that we have working JDK?

Reply:
It should be fine now, although it probably does not have any practical value.

Reply:
@cthbleachbit Please drop `__LOONGSON3` case.